src/Compilers/CSharp/Test/Semantic/Semantics/SwitchTests.cs

```diff
@@ -915,6 +916,47 @@ public static void Main()
             comp.VerifyDiagnostics();
         }
 
+        [Fact]
+        public void MultipleLabesWithBadConstantValues()
+        {
+            var source = @"
+class Program
+{
+    enum E 
+    { 
+        V1 = 1000, 
+        V2 = 1001
+    }
+
+    public static void Main() { } 
+
+    public static void Test(E x)
```

Comment:
@leppie no.  In this case there are explicitly no references including no reference to mscorlib. That makes it impossible for the compiler to interpret the numeric constants because the `int` type doesn't exist and hence they are labelled bad